src/viam/components/camera/camera.py

```diff
@@ -52,6 +52,20 @@ async def get_point_cloud(self, *, timeout: Optional[float] = None, **kwargs) ->
         should encode the bytes into the formatted suggested
         by the mimetype.
 
+        To deserialize the returned information into a numpy array, use the Open3D library.
+        ::
+
+            import numpy as np
+            import open3d as o3d
+
+            data, _ = await camera.get_point_cloud()
+
+            # write the point cloud into a temporary file
```

Comment:
You can probably use a `BytesIO` buffer rather than writing out to a file in case there are permissions issues with writing to disk

Reply:
unfortunately the api only takes file name as input arg https://github.com/isl-org/Open3D/issues/1146